src/modules/user/dto/create-enrolled.dto.ts

```diff
@@ -8,16 +9,45 @@ interface video {
 
 export class CreateEnrolledDTO {
   @IsNotEmpty()
+  @ApiProperty({
+    type: mongoose.Schema.Types.ObjectId,
+    description: 'The id of the student',
+    default: '',
+  })
   studentId: mongoose.Schema.Types.ObjectId;
 
+  @ApiProperty({
+    type: [Boolean],
+    description: 'The description of the student',
+    default: '',
+  })
   videosWatched: boolean[];
```

Comment:
Description is not matching the property.

Reply:
Fixed